query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/insert/Insert.java

```diff
@@ -44,4 +44,24 @@
    */
   @NonNull
   Insert usingTimestamp(@Nullable BindMarker bindMarker);
+
+  /**
+   * Adds a USING TTL clause to this statement with a literal value. Setting a value of {@code 0}
+   * will clear a previously set TTL.
+   *
+   * <p>If this method or {@link #usingTtl(BindMarker) } is called multiple times, the value from
+   * the last invocation is used.
+   */
+  @NonNull
+  Insert usingTtl(int ttl);
+
+  /**
+   * Adds a USING TTL clause to this statement with a bind marker. Setting a value of {@code 0} will
+   * clear a previously set TTL.
+   *
+   * <p>If this method or {@link #usingTtl(int) } is called multiple times, the value from the last
+   * invocation is used.
+   */
+  @NonNull
+  Insert usingTtl(@Nullable BindMarker bindMarker);
```

Comment:
Again I would include an `@param` tag and specify that the bind marker should be understood in seconds.